### ISSUE TITLE
make-pot pointing to raw JSX file, not built JS file

### ISSUE THREAD

**Issue by cliffordp** [wp-cli/i18n-command#212]
`wp i18n make-pot . languages/rankbear.pot --include='dist,src' --exclude='*.jsx,*.map,*.svg,*.txt' --color'`

Example built file (currently private project but will be public) in my *dist* directory:
![image](https://user-images.githubusercontent.com/1812179/80940731-43cfc980-8da6-11ea-91e1-624147f64bd1.png)

Why does the .pot file point to the JSX file instead of the built JS file?
![image](https://user-images.githubusercontent.com/1812179/80941876-1fc1b780-8da9-11ea-846d-e559abdb028a.png)

**Comment by ocean90** [comment]
Sounds like a duplicate of #185.

**Comment by cliffordp** [comment]
Thanks, @ocean90. I tried `--debug` and EVERY file is still in there even though excluded. I assumed this meant it was able to *find* it but then it must have known to exclude it once found (such as excluded by file extension `*.jsx`).

If that's not what debug's output is telling me, then the *exclude* isn't working, I guess being overridden by *include*, which seems odd.

If that's the case, it's not quite the same as #185, since I don't have a wildcard *include*, only a wildcard *exclude*.

Thoughts?

**Comment by ocean90** [comment]
Unfortunately, include is broken too, see #186.

**Comment by cliffordp** [comment]
Interesting. TYVM for the help.

Any idea how bugs in this CLI command affect ability to translate on .org, such as if my plugin has JSX that builds to JS and does not ship JSX files to .org, and I don't generate a .pot for .org, then things should still be translatable by .org's team because all they'd know about is the built JS files, right?

**Comment by ocean90** [comment]
Correct, the built files are what really matters. For testing you can use `wp i18n make-pot .` on the final plugin files and check that everything is included in the POT file.

On w.org we're still using 2.2.0 fwiw.

**Comment by cliffordp** [comment]
gracias!
